### PR TITLE
GBP Currency Converter

### PR DESCRIPTION
**Pull request by Scott-Crawley**

### CODE REVIEW

File: converters/convertGBP.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+// Exchange rates
+const RATES = {
+    USD: 1.40, 
+    EUR: 1.15,
+    RUB: 104.27
+};
+
+// Conversion function
+function convertToGBP(value, fromCurrency) {
```
Comment (umaimahyusuf) [inline converters/convertGBP.js:10]: good name for function

**Review by umaimahyusuf — APPROVED** [review]
good work

File: converters/convertGBP.js (hunk 1 above)
Comment (Scott-Crawley) [reply converters/convertGBP.js:10]: tyvm that means a lot to me!!!!!!!111!!

**Review by EthanM1021 — COMMENTED** [review]
Lovely

File: converters/convertGBP.js (hunk 1 above)
Comment (missKatiaPunter) [reply converters/convertGBP.js:10]: I hope you have some more meaningful conversations guys! 😄 @Scott-Crawley @umaimahyusuf

**Pull request merged**